### ISSUE TITLE
Support for model H6154

### ISSUE THREAD

**Issue by TevanieP** [bwp91/homebridge-govee#64]
* **Please explain your feature request in a one or two sentences.**

I would really appreciate support for Govee device RGB LED Strip Lights H6154 when and if the API is updated. 

* **Is your feature request related to a problem? Please describe.**

Cant register my device to Govee homebridge plugin. 

* **Any particular Govee devices that this relates to?**

Yes, Govee device H6154 (RGB LED Strip Lights)

* **Anything else?**

**Comment by bwp91** [comment]
Hi @TevanieP 
Is this a wifi model? If so, yes there is a chance in the future the API will support it. If it's just bluetooth then the chances are slim.
This plugin doesn't do any 'checking' with supported models, it just works with the data that is provided by Govee. Meaning, as and when the API is updated with new models they should automatically start working with the plugin.
I hope that makes sense!

**Comment by TevanieP** [comment]
Yes, it's a wifi and bluetooth model. I spoke with Govee and they stated that it should be added to the API soon. Thank you for your response, I really appreciate it!

**Comment by bwp91** [comment]
There is a govee 'server upgrade' tomorrow (not sure if you saw the notice in the govee app) so...

> I spoke with Govee and they stated that it should be added to the API soon

Will _hopefully_ be very soon!

**Comment by TevanieP** [comment]
Govee has finally added support for H6154 and many other models in their API update (1.2) 😃 , device is shown as offline though.

**Comment by bwp91** [comment]
Will have a look at this shortly!

**Comment by bwp91** [comment]
Please try the current beta version of the plugin

**Comment by bwp91** [comment]
Make sure it's v3.1.5-beta.7 you're updated to.

**Comment by TevanieP** [comment]
Just realized there was a new beta, sorry about that! Thank you so much!!!

**Comment by bwp91** [comment]
All working on v3.2.2?

**Comment by TevanieP** [comment]
Yep, actually working better than before. Really appreciate the work, will be sponsoring you on Github.

**Comment by bwp91** [comment]
Out of interest how did you realise there was an update to the API? Just by checking the online pdf? I check it on a regular basis but you seemed to beat me on this one!

I didn't know if Govee maybe posted something on their app forums.

**Comment by TevanieP** [comment]
When I got my Govee device and tried to register it with Homebridge-Govee, I quickly realized that my model wasn’t supported. So I emailed the API team and called them to get some traction on the issue; they gave me a date of May 10th, when the API will supposedly be updated. I went to my Homekit app to turn on my other lights that ARE supported, and I saw my LED Strips come up but say “Not Supported” So that's when I found out they finally updated it and I reopened this issue.  :)
